test/integration_run_test.go

```
@@ -0,0 +1,91 @@
+		{
+			name: "NoAll",
+			cmd:  "terragrunt run apply --non-interactive --working-dir " + rootPath,
+		},
+		{
+			name: "AllNoGenerate",
+			cmd:  "terragrunt run apply --all --no-stack-generate --non-interactive --working-dir " + rootPath,
+		},
+	}
+
+	// Run each test case and verify stack generation is skipped
+	for _, tt := range testdata {
```

golangci-lint (1.64.8)

Range statement for test TestRunNoStacksGenerate missing the call to method parallel in test Run


(paralleltest)